service/README.md

```diff
@@ -48,6 +48,21 @@ You should get a response similar to this:
   {"status":"UP","diskSpace":{"status":"UP","total":249644974080,"free":137188298752,"threshold":10485760}}
 ```
 
+Ensure variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/import-ccd-definition.sh 
+is set to http://fpl-service:4000 - this ensures the CCD use the docker networking to reach 
+the service.  
+
+### Run application from IntelliiJ
+For local profile - configure the notify.api_key if necessary (in service/src/main/resources/application.yaml). 
+ 
+Ensure the Spring Boot is started with local profile 
+(add environment variable spring.profiles.active=local when starting the main class).
+
+Change the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/import-ccd-definition.sh to 
```

Comment:
Bit consign, it says change variable but in fact `http://docker.for.mac.localhost:4000` is set to be default one.

It should be really other way around. This paragraph should say `ensure` while similar one in section above should say `change`.

Reply:
Thanks, makes sense!